### PR TITLE
Wednesday September 15 2021 Meeting

### PR DESCRIPTION
**Pull request by nkohen**
Please comment on this PR or DM with things we should discuss or other changes that should be made to this document.

Once the meeting has happened I will add meeting notes and then once they have been reviewed this will be merged into meetings directory.

### CODE REVIEW

File: meeting/meeting_september_15_2021.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+* rust-dlc
+* atomic finance
+* Binary outcome DLC implementation from Lloyd
+* others?
+
+## Specification Writing
```
Comment (Tibo-lg) [inline meeting/meeting_september_15_2021.md:44]: Would be nice to discuss the lexicographical ordering thing

File: meeting/meeting_september_15_2021.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+**#Status_Update_Interrupt**
+
+* If anyone is interested it would be nice to document loss-of-fund vectors
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/132
+  * Being robust against random failures?
+    * Nothing in specification about what should be backed up when before sending messages
+    * In LN they also have the channel-reestablish message
+* Draft Updating P2P message serialization to include sub-types, optional fields, tlv streams
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/171
+  * Closer to TLVs than the previous draft
```
Comment (Tibo-lg) [inline meeting/meeting_september_15_2021.md:55]: Actually I'd like to discuss which one of the two drafts we should go for. After implementing something a bit close to the TLV version one, I'm not really sure it's the best approach.

File: meeting/meeting_september_15_2021.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+* **#Query(Antoine)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+* rust-dlc
+* atomic finance
+* Binary outcome DLC implementation from Lloyd
+* others?
```
Comment (ariard) [inline meeting/meeting_september_15_2021.md:42]: dlc support in rust-lightning? Tibo has done the first minimal step :)

**Comment by LLFourn** [comment]
One thing I thing I'd discuss is the general format of the oracle public key list kind of data structure and perhaps signing it with the main "announcement" key. Or whether we even need such a thing. Perhaps the attestation keys could be in the announcement for example and there is only the announcement key up front.

**Pull request merged**